packages/api-client/tests/secret.spec.ts

```diff
@@ -58,7 +59,8 @@ describe('Secret Controller Tests', () => {
       )
     ).json()) as any
 
-    environmentSlug = createEnvironmentResponse.slug
+    environment = createEnvironmentResponse
+    environmentSlug = environment.slug
```

Comment:
Maybe we can get rid of `environmentSlug` and keep just `environment: Environment`

Reply:
so remove environmentSlug and just use environment.slug in related segements?

Reply:
yes